pyrevitlib/pyrevit/revit/db/query.py

```diff
@@ -798,7 +798,7 @@ def get_category(cat_name_or_builtin, doc=None):
                 return cat
     elif isinstance(cat_name_or_builtin, DB.BuiltInCategory):
         for cat in all_cats:
-            if cat.Id.IntegerValue == int(cat_name_or_builtin):
+            if cat.Id == cat_name_or_builtin:
```

Comment:
`BuiltInCategory` is an enum and not an `ElementId`, so here it should be `value_func(cat.Id) == int(cat_name_or_builtin)`